package.json

```diff
@@ -116,7 +116,7 @@
     "is-stream": "1.1.0",
     "is-svg": "3.0.0",
     "libdweb": "mozilla/libdweb.git#1ee7977a2352950ecf7fa46a13ae45aa500c9b32",
-    "libp2p-webext-mdns": "github:alanshaw/js-libp2p-webext-mdns#661e03745450c3303e4fa0c2600c28dd0a4a3ba5",
+    "libp2p-webext-mdns": "alanshaw/js-libp2p-webext-mdns.git#fake-announce",
```

Comment:
I think I've found alternative notation that works around yarn cache issues:

https://github.com/ipfs-shipyard/ipfs-companion/blob/3e1fd759cab81eddbdab914b23496cb77742b447/package.json#L117

(`185290fb35dc2846e7f31b0a52d0afbb412ab7b5` is the specific git revision you want to use)